Cargo.toml

```diff
@@ -13,21 +13,18 @@ doctest = false
 [features]
 unstable = [
     "string_cache/unstable",
-    "tendril/unstable",
     "rc/unstable",
     "html5ever/unstable",
     "selectors/unstable",
 ]
-with-hyper = ["hyper"]
```

Comment:
I'm a bit under the weather, but here are some of my thoughts. I hope they aren't too incoherent or rambly. 

I think `from_http` would possibly be better as an extension trait in `kuchiki`.

Why I asked about moving `hyper` into `html5ever`? Well my thoughts went like this: html5ever should do one thing best - html parsing. Details of connecting to pages should be left to higher abstraction libraries, like `kuchiki` or `servo`, etc. However I do see merit to `html5ever` having the ability to download an html page to parse itself.

Removing `Html` was a good call if no one but html5ever was going to use it. I like the changes you did to simplify the interface.

I also have to ask - `kuchiki::traits` seem to take `TendrilSink` from `html5ever` instead of `tendril`? Is it because its WIP or because you don't want to depend on tendril or something?